app/helpers/spotlight/title_helper.rb

```diff
@@ -14,7 +14,14 @@ def configuration_page_title(title = nil)
 
     def page_title(section, title = nil)
       set_html_page_title(t(:'spotlight.html_admin_title', section: section, title: title || t(:'.title', default: :'.header')))
-      content_tag(:h1, safe_join([section, content_tag(:small, title || t(:'.header'))], "\n"), class: 'page-header')
+      content_tag(
+        :header,
+        safe_join(
+          [content_tag(:h1, section),
+           content_tag(:h2, title || t(:'.header'))]
```

Comment:
Hi @dunn, I passed this by our UX/a11y knowledgable folks in connection w/ the work happening in sul-dlss/exhibits#1601.  It seems like the content that is in a `<small>` was intended to be part of the `<h1>` and not a separate heading.

It does appear that other heading structure changes will be required in that admin dashboard to make the other heading levels in the dashboard more appropriate (and that work is should be ready for review tomorrow).

In the meantime, if the addition of the `<header>` is important, can this PR be updated to keep the title content in a `<small>` in the `<h1>`?

Reply:
looking at this again, since `page_title` is used for subsections as opposed to the primary page headings, i'm not sure if `<header>` is even appropriate.  since there's existing work on semantics, i'll just close this.

Reply:
Hopefully #2425 will improve things on this front.